src/Apps/Search/Components/NavigationTabs.tsx

```diff
@@ -93,13 +89,16 @@ export class NavigationTabs extends React.Component<Props> {
 
     const tabCountMap = Object.entries(TAB_NAME_MAP).reduce(
```

Comment:
This function looks ideal for covering with unit tests. I _think_ I understand the changes you made, and why you made them, but `array.reduce` operations are (1) easy to mess up, and (2) easy to write tests for.

Reply:
Hey just updated these tests after our pairing session, should be good to merge!